indico/htdocs/css/events/meetings/Intelum.css

```diff
@@ -0,0 +1,29 @@
+/*
+* -*- mode: text; coding: utf-8; -*-
+
+
+   This file is part of Indico.
+   Copyright (C) 2002 - 2014 European Organization for Nuclear Research (CERN).
```

Comment:
please run `python bin/maintenance/update_header.py indico/htdocs/css/events/meetings/Intelum.css` to get the proper header.